tests/test_utils.py

```diff
@@ -1,15 +1,26 @@
 # -*- coding: utf-8 -*-
+import datetime
```

Comment:
NP: Imports should be grouped into system, third-party, and local source tree each of which are separated by a single empty line.